docs/developer-docs/integrations/sns/tokenomics/sns-checklist.md

```diff
@@ -57,46 +58,28 @@ It may not be possible to fully decentralize the dapp at the time of the SNS lau
+Use the [sns cli tool](https://github.com/dfinity/ic/tree/master/rs/sns/cli) to validate the consistency of the configuration file, and test the configuration extensively locally before the final canisters are set up. See section 2.4 for more information about testing.
 
 ## 1.5. Create NNS proposals
-As a part of the SNS launch process, two proposals must be created. The first proposal is for allowing a principle to create the SNS canisters, see Hot or Not’s [command](https://forum.dfinity.org/t/help-submit-the-sns-w-wallet-principal-id-to-be-whitelisted/20322/20) to create the first proposal.
-
-<!-- NEW: As a part of the SNS launch process, an NNS proposals must be created. You can find the details about how to create such a proposal [here](../launching/launch-summary-1proposal.md).-->
-
-
-### 1.5.1. SNS canister creation proposal
-The canister creation proposal is asking the NNS neurons for acceptance to start the SNS launch process, and to allow only the accepted principal to install an SNS.
+<!-- OLD: As a part of the SNS launch process, two proposals must be created. The first proposal is for allowing a principle to create the SNS canisters, see Hot or Not’s [command](https://forum.dfinity.org/t/help-submit-the-sns-w-wallet-principal-id-to-be-whitelisted/20322/20) to create the first proposal.-->
 
-This proposal is not going in-depth with the tokenomics, governance etc. but links to the dapp and a whitepaper should be shared, so the community can evaluate if launching a SNS for the dapp makes sense. See OpenChat’s [proposal 1](https://dashboard.internetcomputer.org/proposal/108634) for inspiration.
+ As a part of the SNS launch process, an NNS proposal must be created to ask the NNS to create an SNS and start a decentralization swap. You can find the details about how to create such a proposal [here](../launching/launch-summary-1proposal.md).
 
-<!-- NEW: REMOVE-->
+Include relevant information like tokenomics (token distribution, governance, decentralization swap), details about the dapp (link to the open sourced code), whitepaper and anything else relevant for the community to make a decisions if an SNS should be launched for the dapp. 
```

Comment:
I suggest to also explicitly mention the link to the deliberation forum post.

Reply:
This actually only comes later (section 3.2.) I can add a forward link, but in the first few sections it is not at all discussed where all of this information is shared. 
Maybe we should consider changing that and do this in a separate MR?

Reply:
I added a forward link to the forum and mentioned the forum discussion. 
I propose that we make further improvements in separate PRs.